tubular/scripts/frontend_utils.py

```diff
@@ -127,6 +153,17 @@ def create_version_file(self):
         except IOError:
             self.FAIL(1, 'Could not write to version file for app {}.'.format(self.app_name))
 
+    def copy_js_config_file_to_app_root(self, app_config, app_name):
+        """ Creates a copy of env.config.js(x) file to the app root """
+        source = app_config['JS_CONFIG_FILEPATH']
+        destination = f"{app_name}/{app_config['LOCAL_CONFIG_FILENAME']}"
```

Comment:
Thanks for getting this work added in here. I'm wondering about this `LOCAL_CONFIG_FILENAME` field. I see it is being set to `webpack.prod.config.js` in the example. Does this mean the JS file will be copied to the `webpack.prod.config.js` file?

For JS file configuration to work properly, we would want this file to be copied to the root of the MFE. That being said, is there a desire/need to copy configuration into the webpack file?